projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -108,4 +110,43 @@ export class NavigationUIComponent implements OnDestroy {
       this.subscription.unsubscribe();
     }
   }
+
+  private alignWrapperToRightIfStickOut(node: HTMLElement) {
```

Comment:
maybe we should think about shortening this name - `alignWrapperToRightIfStickOut()`

Reply:
I don't have a better idea for a name that describes what this function does